doc/tutorials/intro.rst

```diff
@@ -38,14 +38,14 @@ follows.
 .. plot::
     :context: close-figs
 
-    >>> rs = np.random.RandomState(42)  # Reproducible results.
-    >>> W = rs.uniform(size=(30, 30))  # Full graph.
+    >>> rng = np.random.default_rng(1)  # Reproducible results.
```

Comment:
A big switch from the seed 42 to the seed 1. :-)

Reply:
That's an example where the seed 42 was giving a connected graph with `RandomState` but a disconnected one with `default_rng`. Hence I changed it to 1, which got me a connected graph.